package.json

```diff
@@ -17,7 +18,8 @@
     }
   },
   "scripts": {
-    "test": "./node_modules/.bin/standard && ./node_modules/.bin/mocha test"
```

Comment:
Heyo! I'm all for a lint command but linting is our first step in testing?
What's the thinking for a separate lint command?